workshop/pre-work/README.md

```diff
@@ -217,41 +170,22 @@ Alternatively, you can use the IBM Cloud Console to generate the IBM Cloud API k
 
 This is the *api_key* value that you will need to provide in your Jupyter notebooks for accessing the Watson Machine Learning service instance.
 
-### Get the Watson Machine Learning service instance location
-
-<!-- TODO: In case we can only use Frankfurt and Dallas, remove the rest. -->
-**Option 1:**
```

Comment:
They need to fill this in for deploying to the right place in the notebook iirc. Can you check where we fill this in the notebook? I can't remember off the top of my head.

Reply:
In the `machine-learning-in-jupyter-notebook`. We aren't doing that module.

Reply:
Ah right! cool.